index.js

```diff
@@ -102,10 +103,10 @@ module.exports = (
   };
 };
 
-function handleAntdInServer(config, options) {
+function handleAntdInServer(config, options, target) {
   if (!options.isServer) return config;
 
-  const ANTD_STYLE_REGX = /antd\/.*?\/style.*?/;
+  const ANTD_STYLE_REGX = target === 'serverless' ? /(antd\/.*?\/style).*(?<![.]js)$/ : /antd\/.*?\/style.*?/;
```

Comment:
so, `/(antd\/.*?\/style).*(?<![.]js)$/` can use for non-serverless?

Reply:
Maybe :D 
I have only 3 products in production with NextJS, and only one of them has enabled CSS Modules (the serverless one). And the extended RegEx doesn't affect Antd styles, functionality, or anything else. I assume that everything will be ok.